submissions/onyemeMichael/javaScript/script.js

```diff
@@ -0,0 +1,55 @@
+let productList = [
+    {
+        productName: "QUBO P19 5.5",
+        imageUrl: " ../html/image/qubo.jpg",
+        productPrice: "#35,000",
+    },
+
+    {
+        productName: "Samsung Galaxy A21s 6.5",
+        imageUrl: "../html/image/Samsung.jpg",
+        productPrice: "#88,000"
+    },
+
+    {
+        productName: "Apple IPhone 11 Pro Max",
+        imageUrl: "../html/image/i-phone.jpg",
+        productPrice: "#580,000" 
+    }
+
+];
+
+//Using For Loop
+for (let product = 0; product < productList.length; product++) {
```

Comment:
This statement should be less than or equal to @MYKEL7

Reply:
@ToluAkin  this gives me an error.